packages/bot-web-ui/src/app/app.jsx

```diff
@@ -27,7 +27,7 @@ const App = ({ passthrough }) => {
 
     React.useEffect(() => {
         showDigitalOptionsMaltainvestError(core.client, common);
-    }, [core.client.is_options_blocked]);
+    }, [core.client, showDigitalOptionsMaltainvestError, common]);
```

Comment:
shouldn't it still be `core.client.is_options_blocked` in the dependency. I think it should rerun when `is_options_blocked` changes.

Reply:
Let's just change it back to the previous dependencies and just ignore the warning using eslint-disable.